### PR TITLE
git-imerge: new package

### PR DESCRIPTION
**Pull request by spwhitt**

### CODE REVIEW

File: pkgs/applications/version-management/git-and-tools/git-imerge/default.nix [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+stdenv.mkDerivation rec {
+  name = "git-imerge-${version}";
+  version = "0.7.0";
+
+  src = fetchFromGitHub {
+    repo = "git-imerge";
+    owner = "mhagger";
+    rev = version;
+    sha256 = "09czjxgjbby54jx1v5m825k87v8g9g374hwv0r6ss48kv1ipvakq";
+  };
```
Comment (domenkozar) [inline pkgs/applications/version-management/git-and-tools/git-imerge/default.nix]: Please use a tarball for released versions such as https://github.com/mhagger/git-imerge/archive/0.7.0.tar.gz

**Comment by peti** [comment]
@jwiegley, why did you merge this patch even though there was an outstanding change request from @domenkozar?

**Comment by jwiegley** [comment]
@peti Somehow, I didn't see that; this was a mistake merge.

**Comment by jwiegley** [comment]
Commit has been reverted.  @spwhitt Please resubmit your PR with above requested change from @domenkozar.

**Pull request merged**